app/src/main/res/layout/dialog_new_lib.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingBottom="15dp"
+    android:paddingLeft="10dp"
+    android:paddingRight="10dp"
+    android:paddingTop="10dp" >
+
+    <EditText
+        android:id="@+id/new_lib_name"
+        android:hint="@string/name_hint"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/edit_text_holo_light"
+        android:textSize="@dimen/dialog_msg_txt_size"
+        android:inputType="text" />
+
+    <EditText
+        android:id="@+id/new_lib_description"
```

Comment:
better to remove the element, like seahub, we don\`t ask user to input descriptions any more.